generator/src/main/java/com/scottlogic/deg/generator/Visualise.java

```diff
@@ -83,8 +82,7 @@ public void run() {
             new RowSpecMerger(fieldSpecMerger),
             new ConstraintReducer(new FieldSpecFactory(), fieldSpecMerger));
 
-        final List<DecisionTree> treePartitions = new NoopTreePartitioner()
-                .splitTreeIntoPartitions(mergedTree)
+        final List<DecisionTree> treePartitions = Stream.of(mergedTree)
```

Comment:
If you're removing partitioning then there will only be one treePartition, you dont need a collection of them.

See later in this class where there is logic related to when multiple partitions exist, at which point multiple files are created.